src/applications/mhv-medical-records/tests/containers/LandingPage.unit.spec.jsx

```diff
@@ -86,27 +86,17 @@ describe('Landing Page', () => {
 
   it('displays features when feature flags are true', () => {
     const customState = {
-      drupalStaticData: {
-        vamcEhrData: {
-          loading: false,
-        },
-      },
+      drupalStaticData: { vamcEhrData: { loading: false } },
       featureToggles: {
-        loading: false,
-        // eslint-disable-next-line camelcase
-        mhv_medical_records_display_conditions: true,
-        // eslint-disable-next-line camelcase
-        mhv_medical_records_display_labs_and_tests: true,
-        // eslint-disable-next-line camelcase
-        mhv_medical_records_display_notes: true,
-        // eslint-disable-next-line camelcase
-        mhv_medical_records_display_vaccines: true,
-        // eslint-disable-next-line camelcase
-        mhv_medical_records_display_vitals: true,
-        // eslint-disable-next-line camelcase
-        mhv_medical_records_display_settings_page: true,
-        // eslint-disable-next-line camelcase
-        mhv_medical_records_update_landing_page: true,
+        loading: false, // eslint-disable-next-line camelcase
+        mhv_medical_records_display_conditions: true, // eslint-disable-next-line camelcase
+        mhv_medical_records_display_labs_and_tests: true, // eslint-disable-next-line camelcase
+        mhv_medical_records_display_notes: true, // eslint-disable-next-line camelcase
+        mhv_medical_records_display_vaccines: true, // eslint-disable-next-line camelcase
+        mhv_medical_records_display_vitals: true, // eslint-disable-next-line camelcase
+        mhv_medical_records_display_settings_page: true, // eslint-disable-next-line camelcase
+        mhv_medical_records_update_landing_page: true, // eslint-disable-next-line camelcase
+        mhv_landing_page_show_share_my_health_data_link: true, // eslint-disable-next-line camelcase
```

Comment:
I see how this might still work but it's a bit unintuitive. A better way to collapse this would be like this:
```
      /* eslint-disable camelcase */
      featureToggles: {
        loading: false,
        mhv_medical_records_display_conditions: true,
        mhv_medical_records_display_labs_and_tests: true,
        mhv_medical_records_display_notes: true,
        mhv_medical_records_display_vaccines: true,
        mhv_medical_records_display_vitals: true,
        mhv_medical_records_display_settings_page: true,
        mhv_medical_records_update_landing_page: true,
        mhv_landing_page_show_share_my_health_data_link: true
      },
      /* eslint-enable camelcase */
```

Reply:
fixed